src/components/Editor/SearchBar.js

```diff
@@ -354,6 +354,38 @@ const SearchBar = React.createClass({
     e.stopPropagation();
     e.preventDefault();
 
+    const {
+      symbolSearchEnabled,
+      symbolSearchResults,
+      selectedResultIndex,
+    } = this.state;
+    if (symbolSearchEnabled) {
+      const searchResults = symbolSearchResults,
+        resultCount = searchResults.length;
+
+      if (rev) {
+        let nextResultIndex = Math.max(0, selectedResultIndex - 1);
+
+        if (selectedResultIndex === 0) {
+          nextResultIndex = resultCount - 1;
+        }
+        this.setState({ selectedResultIndex: nextResultIndex });
+        this.onSelectResultItem(searchResults[nextResultIndex]);
+      } else {
+        let nextResultIndex = Math.min(
+          resultCount - 1,
+          selectedResultIndex + 1
+        );
+
+        if (selectedResultIndex === resultCount - 1) {
+          nextResultIndex = 0;
+        }
+        this.setState({ selectedResultIndex: nextResultIndex });
+        this.onSelectResultItem(searchResults[nextResultIndex]);
+      }
```

Comment:
Hmm  - I'm a bit confused by this logic. 

It looks very similar to what we're doing in search search to determine the next index. Is there an easy way we can unify it?